pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "openapi-python-client"
-version = "0.7.3"
+version = "1.0.0"
```

Comment:
We should add a comment that our versions have diverged and are maintained internally with no relation to the upstream.

Reply:
Added a small note in regards to version divergence :)

Reply:
Added sed statement to rename package on setup.py and also updated README to identify the change and needing to install/download GNU sed instead of BSD sed.